cmd/bb_copy/BUILD.bazel

```diff
@@ -20,18 +21,43 @@ go_library(
     ],
 )
 
-go_binary(
-    name = "bb_copy",
-    embed = [":bb_copy_lib"],
-    pure = "on",
-    visibility = ["//visibility:public"],
-)
+[
+    go_binary(
+        name = "bb_copy-{}".format(goarch),
+        embed = [":bb_copy_lib"],
+        pure = "on",
+        goarch = goarch,
+        goos = "linux",
+        visibility = ["//visibility:public"],
+    )
+    for goarch in ["amd64", "arm64"]
+]
```

Comment:
I don't think this change is acceptable for two reasons:

1. We no longer build Go binaries for the host architecture. We still want to have binaries for macOS, Windows, FreeBSD, etc..
2. `oci_image_index()` should just use transitions. That way there would be no need for us to put loops around `go_binary()`, `pkg_tar()`, and `oci_image()`.

Reply:
done, PTAL